src/rapids_dependency_file_generator/schema.json

```diff
@@ -162,22 +163,16 @@
         "extras": {
             "type": "object",
             "properties": {
+                "section": {
```

Comment:
Does the keyword "section" originate from the [conda docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section)? I am wondering whether it would make sense to better namespace our "extras", something like "conda-meta-section" instead of just "section".